packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -155,6 +155,20 @@ export interface DropdownProps
   /** A message to be displayed in the list when the dropdown has no items. */
   noResultsMessage?: ShorthandValue<ListItemProps>
 
+  /**
+   * Called when the dropdown's selected items index change.
+   * @param event - React's original SyntheticEvent.
+   * @param data - All props and the new selected value(s).
+   */
+  onActiveSelectedIndexChange?: ComponentEventHandler<DropdownProps>
```

Comment:
Can we suggest in the comment if the event is null for some handler?

Reply:
As far as I see, the event `might` be `null`, depending from which place the handler was called. And not only for this one, also for `onChange` as well. What do you suggest?

Reply:
But we already have an issue regarding this: https://github.com/microsoft/fluent-ui-react/issues/2318 We should distinguish between the component event handlers that have event and the ones which do not. At least this is my opinion, we can ask other people for input